icu4j/main/classes/core/src/com/ibm/icu/impl/units/ComplexUnitsConverter.java

```diff
@@ -23,8 +23,8 @@
  * instances of the `UnitConverter` to perform the conversion.
  */
 public class ComplexUnitsConverter {
-    private static final BigDecimal EPSILON = BigDecimal.valueOf(Math.ulp(1.0));
-    private static final BigDecimal EPSILON_MULTIPLIER = BigDecimal.valueOf(1).add(EPSILON);
+    public static final BigDecimal EPSILON = BigDecimal.valueOf(Math.ulp(1.0));
+    public static final BigDecimal EPSILON_MULTIPLIER = BigDecimal.valueOf(1).add(EPSILON);
```

Comment:
@ Shane :
These are public because they're used from:
src/com/ibm/icu/dev/test/impl/UnitsTest.java
I don't know whether that really necessitates having these public members?